suites/suites/suites.yaml

```diff
@@ -121,6 +121,17 @@ templates:
     <<: *manager_blueprint_commercial_override
     node_templates.manager.properties.cloudify.workflows.task_retries: 100
 
+  - &aws_ec2_manager_blueprint_override
+    <<: *manager_blueprint_override
```

Comment:
`manager_blueprint_override` is good, which is something that we actually forgot initially.